magda-web-client/src/Components/Dataset/DatasetPage.tsx

```diff
@@ -142,6 +143,20 @@ const DatasetPage: FunctionComponent<PropsType> = props => {
                             </button>
                         </div>
                     ) : null}
```

Comment:
We probably don't need this button as we have a new UI for it see https://github.com/magda-io/magda/issues/2718

Reply:
Yep this is for ease of testing. I will remove this before marking the PR as ready :smile:

Reply:
👍

Reply:
Removed